src/ray/core_worker/task_interface.h

```diff
@@ -12,6 +13,11 @@ class CoreWorker;
 
 /// Options of a non-actor-creation task.
 struct TaskOptions {
+  TaskOptions() {}
+  TaskOptions(const int num_returns,
```

Comment:
This is probably an instance where we *don'* want `const`. When we're passing primitive types like this by value. Same with the `const uint64_t` below.

Reply:
accepted, will revise.